### ISSUE TITLE
Changing the final layers of an sm.UNet

### ISSUE THREAD

**Issue by ntelo007** [qubvel/segmentation_models#295]
Hi people,

you created a wonderful repository! 

Could you please help me apply this modification? I just want to have a Unet, with ResNet34 architecture, pre-trained with ImageNet and where the final layer of the classifcation is splitted into 3 instead of just 1 layer. 

Thank you in advance.

**Comment by JordanMakesMaps** [comment]
I know you're looking to do multi-task learning, but regarding multiple outputs and multiple loss functions [here's](https://www.pyimagesearch.com/2018/06/04/keras-multiple-outputs-and-multiple-losses/) a tutorial for constructing a network.

**Comment by ntelo007** [comment]
Thank you very much. I will have a look at it!
